pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+## Problem
+Why are we making this code change?
+
+## Solution
+How do the changes in this pull request solve the stated problem? Be descriptive.
+
+## Additional Notes
+Is there anything in particular that you want to call attention to? Areas of focus, follow-up actions, etc.
+
+## Links
+### Ticket
+- [ ] GitHub issue _____
```

Comment:
Can we make this `Task ticket` since the majority of our tasks come from Asana?

Reply:
I leaned heavily into github because I thought we were previously advised against linking to private/internal ticketing systems for OSS

Reply:
added note to the template about not linking to private ticketing systems